submission/persistence/src/main/kotlin/ac/uk/ebi/biostd/persistence/mapping/extended/to/ToExtSubmissionMapper.kt

```diff
@@ -4,21 +4,25 @@ import ac.uk.ebi.biostd.persistence.model.SubmissionDb
 import ebi.ac.uk.extended.model.ExtAccessTag
 import ebi.ac.uk.extended.model.ExtSubmission
 import ebi.ac.uk.extended.model.ExtTag
-import ebi.ac.uk.functions.secondsToInstant
+import ebi.ac.uk.io.sources.ComposedFileSource
+import ebi.ac.uk.io.sources.FilesSource
 import ebi.ac.uk.io.sources.PathFilesSource
+import ebi.ac.uk.model.SubmissionMethod.UNKNOWN
 import java.nio.file.Path
-import java.time.ZoneOffset
 
 private const val FILES_DIR = "Files"
 
+// Added for background compatibility of old submitter applications
```

Comment:
typo: backward compatibility

Reply:
this is a new compatibility type that I just invented

Reply:
:laughing: